zend_abstract_interface/zai_sapi/zai_sapi_ini.h

```diff
@@ -0,0 +1,28 @@
+#ifndef ZAI_SAPI_INI_H
+#define ZAI_SAPI_INI_H
+
+#include <stdbool.h>
+#include <stddef.h>
+
+/* Allocates memory for 'dest' and copies the entires from 'src' into it.
```

Comment:
```suggestion
/* Allocates memory for 'dest' and copies the entries from 'src' into it.
```

Reply:
Wow. I really don't know how to spell "entries". 😆 Thanks for detecting all these!